### ISSUE TITLE
[Feature] option to disable health-info server-side

### ISSUE THREAD

**Issue by albertz** [openlierox/openlierox#767]
As said by Tafka [here](http://www.openlierox.net/forum/index.php/topic,13625.msg235394.html#msg235394), it changes gameplay if the health of other players is unknown and this is even the wanted behavior. So there should at least be an option to get this behavior.

**Comment by karelp** [comment]
I don't think it is that important to consider this. The worms are bleeding when they are low on health (this was the behavior in old LX as well), thus adding health bars didn't really change the gameplay that much. It would mean another option which much less than 1% of users would ever use.

**Comment by albertz** [comment]
Some bleeding at around 15% is much less information than a health bar. There is a huge difference. And I agree on Tafka, that can have some impact on the gameplay.

The option itself will not hurt, so I'm adding it anyway. Not sure yet where in the GUI I will place it - maybe nowhere atm if I don't find a good place.

What I was talking about here - and I think you misunderstood me - is to make the health info disabled by default. I.e. this is then what 99% of the people will be using, not 1%.

**Comment by karelp** [comment]
It's changing the defaults (again). I am not a fan of changing a thing that works and works well. Sure, it affects gameplay but the aim speed change was just like that. Yet, aimspeed remains and health bars should be gone?

Btw., why do you ask when you do it anyway? :)

**Comment by albertz** [comment]
Well, my goal is of course to improve the gameplay. :) I think faster (and also configureable) aimspeed is better (but this is OT here).

And it might be that health-bar disabled is also better. Not sure. Maybe I'll leave it enabled by default for the beginning (i.e. next 0.59 beta release) and if people agree with me/Tafka, I make health-bars disabled by default later. (And for those who use the LX56 settings preset, they will anyway have it disabled by default then.)

**Comment by karelp** [comment]
Ok, that sounds reasonable.
